### PR TITLE
Disable contrail analytics

### PR DESCRIPTION
**Pull request by hkumarmk**
We are not using analytics data at this moment, so it doesnt make sense to
collect all these data especially because contrail collector who collect all the
network flows and other details is a heavy process which cause others to
slowdown. So disabling contrail-analytics for now.

### CODE REVIEW

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
cancelled last test because of a correction to be made in contrail module.

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
last test failed because i missed to remove the validation check for contrail-analytics.

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
last test failed because stmonleader1 was not got userdata executed because of metadata failure.

**Comment by hkumarmk** [comment]
please test this

**Comment by hkumarmk** [comment]
yeey,.... this is also got passed :)

File: Puppetfile [hunk 1]
```diff
@@ -142,8 +142,8 @@ mod 'stephenrjohnson/puppet',
   :ref => '1.0.0'
 
 mod 'jiocloud/contrail',
-  :git => "#{base_url}/jiocloud/jiocloud-contrail",
-  :ref => 'origin/master'
+  :git => "#{base_url}/hkumarmk/jiocloud-contrail",
```
Comment (bodepd) [inline Puppetfile]: this has been merged into master. Can you update this ref and retest?

Comment (bodepd) [inline Puppetfile]: other than that, it looks ready to merge!

**Comment by hkumarmk** [comment]
@bodepd as we discussed, just merging this, I fixed the change you suggested.

**Pull request merged**